raiden/network/proxies/secret_registry.py

```diff
@@ -134,43 +131,34 @@ def register_secret_batch(self, secrets: List[Secret]) -> None:
         # From here on the lock is not required. Context-switches will happen
         # for the gas estimation and the transaction, however the
         # synchronization data is limited to the open_secret_transactions
-        log_details = {
```

Comment:
the `node`, `contract`, and function arguments are now logged by default. Just kept the `secrethashes_not_sent` for additional context.